flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/tasks/InputSplitProvider.java

```diff
@@ -35,4 +35,16 @@
 	 *         task shall not consume any further input splits.
 	 */
 	InputSplit getNextInputSplit();
+
+	/**
+	 * Starts the input split provider with a user code class loader.
+	 *
+	 * @param userCodeClassLoader User code class loader to use by the input split provider
+	 */
+	void start(ClassLoader userCodeClassLoader);
```

Comment:
Is it possible to have the `start()` and `stop()` methods not in the base interface, but only in the Akka specific implementation?